apps/prairielearn/src/pages/partials/question.ejs

```diff
@@ -182,7 +182,7 @@
         $(function() {
           $('.js-submission-body.render-pending').on('show.bs.collapse', function(e) {
             const submissionId = $(this).data('submission-id');
-            // <% if (question_context === 'instructor') { %>
+            // <% if (question_context === 'instructor' || question_context === 'public') { %>
```

Comment:
🤮 (I know this is not related to your change but wow, this pattern of sticking EJS control flow in JS comments is horrible)

Reply:
Ya, this is going to be much nicer once we're all in `.html.ts` land. Unfortunately this particular file is not going to be pleasant to convert.